dashboards-how-to (#6935)
mattermost/docs: source/scale/performance-monitoring.rst

Mattermost Metrics plugin
-------------------------

The Mattermost Metrics plugin, is an alternative tool to collect application metrics from Mattermost. It can be installed on versions later than v6.3, collects and stores the `same performance monitoring metrics <#statistics>`__ as if Prometheus, without having to deploy these third-party tools. Data is collected every minute and is stored where the plugin is running. The data is sychronized to either a cloud-based or local file store every hour, and retained for 15 days. 

Using the Mattermost Metrics plugin, you can download and share the collected data with Mattermost to understand application performance, troubleshoot system stability and performance, as well as inform route cause analysis.

> Furthermore, to use the dump file genereated by the plugin, you can simply clone the `Dockprom <https://github.com/stefanprodan/dockpromo>`__ Repository. Then you should change the prometheus data volume to point the dump that youjust downloaded. Note that the downloaded file is compressed, to be able to use it, you should decmpress it first.

Reviewer: - typo at generated.
- Also space between "youjust".
- Typo at "decompress" as well

> Furthermore, to use the dump file generated by the plugin, you can simply clone the `Dockprom <https://github.com/stefanprodan/dockpromo>`__ Repository. Then you should change the Prometheus data volume to point to the dump that you just downloaded. Note that the downloaded file is compressed, to be able to use it, you should decompress it first.

Reviewer 2: ```suggestion
Furthermore, to use the dump file generated by the plugin, you can simply clone the `Dockprom <https://github.com/stefanprodan/dockpromo>`__ repository. Change the Prometheus data volume to point to the dump that you just downloaded. The downloaded file is compressed, so to be able to use it, you need to decompress it first.
```

> The volume configuration for Prometheus should look like the below in the ``docker-compose.yml`` file:

Reviewer 2: ```suggestion
The volume configuration for Prometheus should look like the code below in the ``docker-compose.yml`` file:
```

.. code:: yaml

    volumes:
      - ./prometheus:/etc/prometheus
      - /Path/To/Dump/Directory:/prometheus/data

> Once you set this up, run ``docker-compose`` as described in `Dockprom Repository <https://github.com/stefanprodan/dockprom?tab=readme-ov-file#install>`__. You can also use our official performance monitoring dashboard by simply importing it into Grafana. To do so, open Grafana (By default it's ``<localhost>:3000``) and then log into it. Once you log it, navigate to the "+" icon on the left sidebar and select "Import". Enter the dashboard ID (``15582``) in the "Grafana.com Dashboard" field and then click "Load" to fetch the dashboard. 

Reviewer 2: ```suggestion
Once you set this up, run ``docker-compose`` as described in `Dockprom Repository <https://github.com/stefanprodan/dockprom?tab=readme-ov-file#install>`__. You can also use our official performance monitoring dashboard by simply importing it into Grafana. To do so, open Grafana (``<localhost>:3000`` by default) and then log into it. Once you log in, go to the **+** icon on the left sidebar and select **Import**. Enter the dashboard ID (``15582``) in the **Grafana.com Dashboard** field and then select **Load** to fetch the dashboard. 
```
